### PR TITLE
Refactor: [M3-6522-selection-card] Selection Card: named exports and v7 story

### PR DESCRIPTION
**Pull request by abailly-akamai**
## Description 📝
This PR refactors the `SelectionCard` component exports and handles the new way of doing stories. It does not modify the component in any way, or at least shouldn't. Part of our epic to migrate away from .mdx files.

## Preview 📷
No visual diff

## How to test 🧪
1. Confirm no regression on the SelectionCard component
2. Confirm and review new story at /?path=/docs/components-selectioncard--documentation

### CODE REVIEW

File: packages/manager/src/components/SelectionCard/CardBase.styles.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  alignItems: 'center',
+  columnGap: theme.spacing(2),
+}));
+
+export const CardBaseSubheading = styled('div', {
+  label: 'CardBaseSubheading',
+})(({ theme }) => ({
+  color: theme.palette.text.primary,
+  fontSize: '0.875rem',
+}));
```
Comment (abailly-akamai) [inline packages/manager/src/components/SelectionCard/CardBase.styles.ts:85]: Nothing new, just putting styles in their own files per out standard practices

**Review by bnussman-akamai — APPROVED** [review]
Looks good! No UI regressions 🎉

**Review by cpathipa — APPROVED** [review]
Nice work!

**Comment by bnussman-akamai** [comment]
Thanks for fixing the lockfile!

**Pull request merged**